content/kubermatic/main/architecture/concept/kkp-concepts/kkp-security/iam/_index.en.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Kubermatic Kubernetes Platform IAM"
```

Comment:
I'm not sure I get why we're creating this page instead of updating the previous parent page for security in some way. This creates an even more complex nested structure to discover this topic. I don't think this is really accessible to people browsing the documentation:

![Screenshot 2023-02-16 at 08 49 05](https://user-images.githubusercontent.com/10295525/219301344-a9999e3e-cfa8-443c-b36d-e7996d04dd1d.png)

Reply:
Yes, I also don't know what value it brings to just putting them into this parental category. But I am curious about @vgramer 's thoughts on that, maybe we miss something.

Reply:
SEO and searchability is the key here for the word IAM. Might not be a bad solution though then.

Reply:
My understanding is that IAM regroups the authentification and authorization strategy, and those topics belong to `KKP security` section. Moreover the `securing system services`  is a related topic and I wanted to keep it close.

However, I agree it's too deep in the documentation structure. As @embik suggest, I can merge it with the  existing `Role based Access Control` page by:

-  renaming the `Role based Access Control` to `KKP IAM`(so it's searchable)
- add the content of the current `Role based Access Control` page under the authorization section 

WDYT ?

Reply:
I have  merged the page together to avoid a nested structure